tables/heb.tbl

```diff
@@ -0,0 +1,260 @@
+
+# liblouis: Hebrew
+#
+# ------------
+#-index-name: Hebrew uncontracted
+#-display-name: Hebrew uncontracted
+#
+#+language:heb
+#+type:literary
+#+contraction:no
+#+system:heb
```

Comment:
From the discussion it seems that using "heb" to identify the implemented braille code is ambiguous. If I understand correctly, it should at least have "IHBC" in it? And perhaps also a reference to McAllester? What do you think? "heb" seems too general.

Reply:
Just wanted to mention that we are working on all your suggested changes for the metadata and we really apreciate the feedback. Hopefully we will push them tomorrow.

Reply:
@paulGeoghegan @yeatersink I pushed some changes in b30358d3. My suggestion for the "system" metadata is `#+system: ihbc-McAllestor` instead of `#+system: hbo`. What do you think?